lms/djangoapps/courseware/views/views.py

```diff
@@ -1061,7 +1063,7 @@ def dates(request, course_id):
     user_timezone = user_timezone_locale['user_timezone']
     user_language = user_timezone_locale['user_language']
 
-    missed_deadlines, enrollment_mode = dates_banner_should_display(course_key, request)
+    missed_deadlines = dates_banner_should_display(course_key, request)
```

Comment:
should we change `dates_banner_should_display` to return only one value?

Reply:
oh, shoot, yeah, I thought I updated the source function to only return one value.

And, I just merged... Need to pause the pipeline.